modules/HardwareDrivers/PowerMeters/DZG_GSH01/slip_protocol.cpp

```diff
@@ -32,6 +35,9 @@ inline bool is_message_crc_correct(std::vector<std::uint8_t>& vec) {
 }
 
 inline void restore_special_characters(std::vector<std::uint8_t>& vec) {
+    if (vec.size() < 1) {
+        return;
+    }
```

Comment:
Looks redundant because the for loop wouldn't execute for `size < 2`
```suggestion
```

Reply:
correct, removed the commit :)